src/core/notifier/handler/webhook/manager.go

```diff
@@ -0,0 +1,169 @@
+package webhook
```

Comment:
do we have to call it manager.go?

Reply:
> do we have to call it manager.go?

I've changed name to `process.go`, any better idea?